docs/src/operate/troubleshooting/monitor_tedge_health.md

```diff
@@ -12,73 +12,78 @@ These services expose MQTT health endpoints which you can query to check if the
 To get the last known health status of a service you can subscribe to the following topic
 
 ```text
-tedge/health/<tedge-service-name>
+te/+/+/+/+/status/health
 ```
 
 To refresh the health status of the service, publish an empty message on the topic below.
 
 ```text
-tedge/health-check/<tedge-service-name>
+te/+/+/+/+/cmd/health/check
 ```
 
 :::note
 If the response is not received then most likely the service is down, or not responding
 :::
 
 
-For example, `tedge-mapper-c8y` publishes below message on topic `tedge/health/tedge-mapper-c8y` when it starts
+For example, `tedge-mapper-c8y` publishes a message on topic `te/device/main/service/tedge-mapper-c8y/status/health` when it starts:
 
 ```json
-{"pid":290854,"status":"up","time":1674739912}
+{ "pid": 290854, "status": "up", "time": 1674739912 }
```

Comment:
The timestamp has changed to RFC3339 format recently.

Reply:
fixed 3b2953620f